lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -376,8 +376,12 @@ protected function getMobileAppDate() {
       $timing = [];
       $timing['dates'] = (dosomething_helpers_extract_field_data($this->node->field_mobile_app_date));
 
-      foreach ($timing['dates'] as $key => $date) {
-        $timing['dates'][$key] = dosomething_helpers_convert_date($date);
+      if ($timing['dates'] != null) {
```

Comment:
does if(isset($timing['dates'])` work here, we tend to use that as the standard check